config/locales/views/events/en.yml

```diff
@@ -27,6 +27,11 @@ en:
     add_price_group: Add price group
     remove_price_group: Remove this price group
 
+    age_limit: Age limit
+    area: Area
+    organizer: Organizer
+    ticket: Ticket
```

Comment:
Oh, they shouldn't appear in no.yml either, as they are only needed in the events.forms.labels, I'll fix it.